plugins/techdocs/src/components/DocsResultListItem/DocsResultListItem.tsx

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import {
+  Box,
+  Chip,
+  Divider,
+  ListItem,
+  ListItemText,
+  makeStyles,
+} from '@material-ui/core';
+import { Link } from '@backstage/core-components';
+
+const useStyles = makeStyles({
+  flexContainer: {
+    flexWrap: 'wrap',
+  },
+  itemText: {
+    width: '100%',
+    marginBottom: '1rem',
+  },
+});
+
+export const DocsResultListItem = ({ result }: any) => {
+  const classes = useStyles();
+  if (result.location.includes('#')) return null;
+  return (
+    <Link to={result.location}>
+      <ListItem alignItems="flex-start" className={classes.flexContainer}>
+        <ListItemText
+          className={classes.itemText}
+          primaryTypographyProps={{ variant: 'h6' }}
+          primary={result.title}
+          secondary={result.text}
+        />
+        <Box>
+          {result.name && (
+            <Chip label={`Entity: ${result.name}`} size="small" />
+          )}
```

Comment:
You also added lifecycle to your document, maybe you want to output it here too.

Reply:
Didn't you choose `entity` as the property name in the interface above? But I would prefer `name`

Reply:
I'm not sure that entity lifecycle value makes sense for a tech docs document. Might need some persuasion before adding that in.

Reply:
Just to throw another idea out there: instead of a chip here, you could implement a common webpage `<title>` pattern and incorporate the entity name into the primary title display, e.g. `Page Title | entity-name`, or if you indexed the `site_name` in the collator, you could use that here for a slightly friendlier name, e.g. `Page Title | Entity Name Docs`

Reply:
Yup, that makes sense for now. I'll modify this to be like so: 
![image](https://user-images.githubusercontent.com/2392775/125085755-39584080-e0cb-11eb-95c6-25dcaac6cb03.png)